From: Hiroaki Nakamura <hnakamur@gmail.com>
Date: Sun, 3 Sep 2017 11:50:41 +0900
Subject: Skip terminal tests in no tty environment like Ubuntu Launchpad

---
 src/testdir/test_terminal.vim | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/src/testdir/test_terminal.vim b/src/testdir/test_terminal.vim
index c7ff89e..57e2e92 100644
--- a/src/testdir/test_terminal.vim
+++ b/src/testdir/test_terminal.vim
@@ -4,6 +4,11 @@ if !has('terminal')
   finish
 endif
 
+" Skip terminal tests in no tty environment like pbuilder and Ubuntu Launchpad
+if $TERM == ""
+  finish
+endif
+
 source shared.vim
 
 let s:python = PythonProg()
