From: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Date: Thu, 31 Aug 2017 19:02:32 +0900
Subject: ubuntu-grub-syntax

===================================================================
---
 runtime/syntax/grub.vim | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/runtime/syntax/grub.vim b/runtime/syntax/grub.vim
index 3743ae3..7d57f44 100644
--- a/runtime/syntax/grub.vim
+++ b/runtime/syntax/grub.vim
@@ -40,7 +40,7 @@ syn keyword grubCommand       contained bootp color device dhcp hide ifconfig
                               \ displaymem embed find fstest geometry halt help
                               \ impsprobe initrd install ioprobe kernel lock
                               \ makeactive map md5crypt module modulenounzip pause
-                              \ quit reboot read root rootnoverify savedefault setup
+                              \ quiet quit reboot read root rootnoverify savedefault setup
                               \ testload testvbe uppermem vbeprobe
 
 syn keyword grubSpecial       saved
