From: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Date: Thu, 31 Aug 2017 19:03:51 +0900
Subject: ubuntu-mouse-off

---
 runtime/defaults.vim | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/runtime/defaults.vim b/runtime/defaults.vim
index 0dcb922..42b983e 100644
--- a/runtime/defaults.vim
+++ b/runtime/defaults.vim
@@ -73,12 +73,6 @@ map Q gq
 " Revert with ":iunmap <C-U>".
 inoremap <C-U> <C-G>u<C-U>
 
-" In many terminal emulators the mouse works just fine.  By enabling it you
-" can position the cursor, Visually select and scroll with the mouse.
-if has('mouse')
-  set mouse=a
-endif
-
 " Switch syntax highlighting on when the terminal has colors or when using the
 " GUI (which always has colors).
 if &t_Co > 2 || has("gui_running")
