From: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Date: Thu, 31 Aug 2017 19:02:47 +0900
Subject: update-upstart-syntax

---
 runtime/syntax/upstart.vim | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/runtime/syntax/upstart.vim b/runtime/syntax/upstart.vim
index 140cd17..8a36ee0 100644
--- a/runtime/syntax/upstart.vim
+++ b/runtime/syntax/upstart.vim
@@ -35,7 +35,7 @@ syn cluster upstartShellCluster contains=@Shell
 " one argument
 syn keyword upstartStatement description author version instance expect
 syn keyword upstartStatement pid kill normal console env exit export
-syn keyword upstartStatement umask nice oom chroot chdir exec
+syn keyword upstartStatement umask nice oom chroot chdir exec setuid setgid
 
 " two arguments
 syn keyword upstartStatement limit
